gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-group/shared-policy-group-history/history-studio-dialog/history-studio-dialog.component.html

```diff
@@ -51,5 +51,18 @@
 </mat-dialog-content>
 
 <mat-dialog-actions align="end">
+  @if (startRestore) {
+    <gio-banner-info class="restore-banner">
+      You are about to apply the previous version.
```

Comment:
`You are bout to apply a previous version`
it is not necessarily the previous one as you can select the version, no ?

Reply:
Well it must be a previous version. not just the -1 🤔

Reply:
maybe `You are about to apply a previous version.`